gradle/code_quality_tasks.gradle

```diff
@@ -29,7 +29,9 @@ task pmd(type: Pmd) {
     // complete rulesets can be used with the abbrevation
     // rulesets with specific excludes are moved to a config file containing just a single ruleset with exclude(s)
 	ruleSets = ['java-android', 'java-braces', 'java-sunsecure', 'java-unusedcode', 'config/pmd_basic.xml',
-				'config/pmd_empty.xml', 'config/pmd_strings.xml', 'config/pmd_unnecessary.xml']
+				'config/pmd_empty.xml', 'config/pmd_strings.xml', 'config/pmd_unnecessary.xml',
+				'config/pmd_logging.xml']
+
```

Comment:
And the last thing I obviously forgot. Convert the tabs to spaces in all previous lines including ```ruleSets = ...``` (but not in the checkstyle task because it's unrelated)